weave-js/src/components/Panel2/availablePanels.tsx

```diff
@@ -170,6 +170,12 @@ function scoreHandlerStack(type: Type, hs: PanelStack) {
   ) {
     scoreHs -= 50;
   }
+
+  // Until combined 2d projection performance is addressed, bump it down in the recommendation list (i.e. don't auto recommend it)
+  if (sidAndName.id.includes('merge.projection.plot')) {
```

Comment:
can you change this to remove the `merge` part - `merge` only occurs if there are multiple tables, but it is possible to just have a single table.